common/djangoapps/track/views/tests/test_segmentio.py

```diff
@@ -404,3 +405,207 @@ def test_video_event(self, name, event_type):
 
         self.assertEqualUnicode(actual_event, expected_event_without_payload)
         self.assertEqualUnicode(payload, expected_payload)
+
+    @data(
+        (1, "onSlideSeek"),
+        (-2, "onSlideSeek"),
+        (30, "onSkipSeek"),
+        (-30, "onSkipSeek")
+    )
+    @unpack
+    def test_old_android_build(self, requested_skip_interval, type):
```

Comment:
should rename 'type' to 'expected_type'.  I'm surprised this wasn't a pylint warning.